src/components/List/components/ListItem.tsx

```diff
@@ -69,7 +71,13 @@ export class ListItem<T = unknown> extends React.Component<ListItemProps<T>> {
         return <div className={b('item-content')}>{renderItem(item, active, itemIndex)}</div>;
     }
 
-    private onClick = () => this.props.onClick?.(this.props.item, this.props.itemIndex);
+    private onClick: React.MouseEventHandler<HTMLDivElement> = (event) => {
+        this._publishEvent({
```

Comment:
Do you need to publish click event in case of handler absence?

Reply:
Yes, i have to publish click event if the item can be clicked.

Reply:
You may have case when item does not have own click handler. Why component should trigger click event in that case?

Reply:
Because we collecting user clicks, not which handlers have been triggered by the user.